lib/utils.js

```diff
@@ -1,24 +1,27 @@
-var version = require('../package.json').version
-var phin = require('phin')
+const version = require('../package.json').version
+const phin = require('phin').unpromisified
 
-var headers = {
+const headers = {
   'User-Agent': 'node-yggdrasil/' + version,
   'Content-Type': 'application/json'
 }
 
-var utils = {}
+const utils = {}
 
 utils.phin = phin
 
 /**
  * Generic POST request
  */
-utils.call = function (host, path, data, cb) {
+utils.call = function (host, path, data, cb, agent) {
```

Comment:
cb must be the last parameter

Reply:
I made cb second to last as i felt uncomfortable with making people set agent to undefined when they don't need to use an agent, but thinking about it now ill just make it detect if the cb is in the agent value and rearrange it.

Reply:
This function is not exposed to users so I don't think you need to handle the case when there is no agent since in all the usages you do pass an agent